.changeset/swift-trees-double.md

```diff
@@ -0,0 +1,5 @@
+---
+"@astrojs/tailwind": patch
```

Comment:
do we need to do a release from this? People will just read this from the github anyways, right?

Reply:
This also updates the README which we usually trigger a patch for but I can remove if you think it's unnecessary.